From: Giovanni Mascellani <mascellani@poisson.phc.unipi.it>
Date: Tue, 16 Aug 2011 14:12:30 +0200
Subject: Do not compile module generator.

Module generator is distributed with a non-free license, so it must
be removed from the Debian package.
---
 pom.xml |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/pom.xml b/pom.xml
index e1d0a50..dd1af49 100644
--- a/pom.xml
+++ b/pom.xml
@@ -60,7 +60,6 @@ EVEN IF SUN HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.
         <module>xsdlib</module>
         <module>testharness</module>
         <module>msv</module>
-        <module>generator</module>
         <module>rngconverter</module>
     </modules>
 
-- 
